### ISSUE TITLE
4.3.8 in docker , some times  crasher: initial call: cowboy_clear:connection_process/4, 

### ISSUE THREAD

**Issue by tangtangsara** [emqx/emqx#6754]
how to resolve it , and why ?

2022-01-14T14:55:01.149524+08:00 [error] fe95a4d4c3501@10.4.0.120:45242 crasher: initial call: cowboy_clear:connection_process/4, pid: <0.29521.98>, registered_name: [], error: {{case_clause,{error,closed}},[{cowboy_websocket,websocket_send_close,2,[{file,"cowboy_websocket.erl"},{line,640}]},{cowboy_websocket,websocket_close,3,[{file,"cowboy_websocket.erl"},{line,623}]},{proc_lib,wake_up,3,[{file,"proc_lib.erl"},{line,236}]}]}, ancestors: [<0.2121.0>,<0.2120.0>,ranch_sup,<0.1569.0>], message_queue_len: 1, messages: [{tcp_closed,#Port<0.22018895>}], links: [<0.2121.0>], dictionary: [{{subscribe,<<"zhibo/room/676">>},{allow,1642143239886}},{{subscribe,<<"zhibo/chat/dm/29668018/#">>},{allow,1642143239889}},{outgoing_bytes,16},{send_oct,16},{guid,{1642143239304084,3802690253649,0}},{incoming_bytes,129},{acl_keys_q,{[{subscribe,<<"zhibo/chat/dm/29668018/#">>}],[{subscribe,<<"zhibo/room/676">>}]}},{send_cnt,4},{acl_cache_size,2},{'$logger_metadata<br class="Apple-interchange-newline">,#{clientid => "fe95a4d4c3501",peername => "10.4.0.120:45242"}},{rand_seed,{#{bits => 58,jump => #Fun<rand.3.47293030>,next => #Fun<rand.0.47293030>,type => exsss,uniform => #Fun<rand.1.47293030>,uniform_n => #Fun<rand.2.47293030>},[186040015765134509\|286552739701653191]}},{recv_pkt,4},{recv_oct,129},{send_pkt,4},{recv_cnt,4}], trap_exit: true, status: running, heap_size: 6772, stack_size: 28, reductions: 152827; neighbours:

2022-01-14T14:55:01.151451+08:00 [error] Ranch listener 'mqtt:ws:8083' had connection process started with cowboy_clear:start_link/4 at <0.29521.98> exit with reason: {{case_clause,{error,closed}},[{cowboy_websocket,websocket_send_close,2,[{file,"cowboy_websocket.erl"},{line,640}]},{cowboy_websocket,websocket_close,3,[{file,"cowboy_websocket.erl"},{line,623}]},{proc_lib,wake_up,3,[{file,"proc_lib.erl"},{line,236}]}]}

**Comment by HJianBo** [comment]
It looks like a bug in the underlying HTTP library(cowboy). We will check it later :)

Thanks for your feedback. @tangtangsara

**Comment by 1995parham** [comment]
We also have this log on websocket listener.

```
emqx 2022-01-18T17:48:32.219573+00:00 [error] Ranch listener 'mqtt:ws:8083' had connection process started with cowboy_clear:start_link/4 at <0.28514.41> exit with reason: {{case_clause,{error,closed}},[{cowboy_websocket,websocket_send_close,2,[{file,"cowboy_websocket.erl"},{line,640}]},{cowboy_websocket,websocket_close,3,[{file,"cowboy_websocket.erl"},{line,623}]},{proc_lib,wake_up,3,[{file,"proc_lib.erl"},{line,236}]}]}

```

**Comment by zmstone** [comment]
Fixed since EMQX v4.4.15
ref:  https://github.com/emqx/cowboy/pull/8
